app/services/health_checks/gov_uk_notify.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module HealthChecks
+  class GovUkNotify
+    URL = 'https://status.notifications.service.gov.uk'
+    COMPONENTS = ['API', 'Email sending'].freeze
+    OPERATIONAL = 'operational'
+    NAME = 'name'
+    STATUS = 'status'
+
+    def healthy?(force: false)
+      Rails.cache.fetch('health_checks_gov_uk_notify_healthy', expires_in: 30.seconds, force: force) do
+        healthy = false
+        begin
+          response = client.get(URL)
+          if response.success?
+            data = JSON.parse(client.get(URL).body)
+
+            # NB this produces a hash e.g. {"API"=>true, "Email sending"=>false}
+            components = data['components']
+                             .select { |c| COMPONENTS.include?(c[NAME]) }
+                             .map { |c| [c[NAME], c[STATUS] == OPERATIONAL] }
+                             .to_h
+
+            # NB: we are healthy if we have checked all the relevant components and they are all true
+            healthy = components.keys.sort == COMPONENTS.sort && !components.values.include?(false)
+            warning(components) unless healthy
+          else
+            warning(response: response.status)
+          end
+        rescue StandardError => e
+          warning(error: e.message)
+        end
+        healthy
+      end
+    end
+
+  private
+
+    def client
+      Faraday.new(headers: { 'Accept': 'application/json', 'User-Agent': 'hmpps-book-a-secure-move/v1' })
+    end
+
+    def warning(details)
+      Rails.logger.warn("[GovUkNotify] service is unhealthy: #{details.to_json}")
+      Raven.capture_message('[GovUkNotify] service is unhealthy', extra: details, level: 'warning')
```

Comment:
what is gonna be the warning level in sentry? 
maybe we need to be explicit we do not want it just as log

Reply:
I tested and these warnings do appear in Sentry. Not sure how it can be more explicit ?

Reply:
(and the warnings appear in slack too:

<img width="467" alt="Screenshot 2020-03-18 at 14 06 29" src="https://user-images.githubusercontent.com/86833/76969039-c220fb00-6921-11ea-96d1-33687c1db35c.png">
)